tests/framework-tests/preferences/js/PanelsTests.js

```diff
@@ -1291,10 +1311,8 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 expect: 2,
                 name: "Test the rendering of the line space panel",
                 sequence: [{
-                    func: "{lineSpace}.refreshView"
-                }, {
                     listener: "fluid.tests.testDefault",
-                    event: "{lineSpace}.events.afterRender"
+                    event: "{lineSpacePanel textfieldSlider}.events.onCreate"
```

Comment:
Same comment as above for running textSizePanel tests at textfieldSlider.onCreate event.